apps/dashboard/src/app/nebula-app/(app)/components/ChatBar.tsx

```diff
@@ -104,10 +105,35 @@ export function ChatBar(props: {
     },
   });
 
-  async function handleImageUpload(images: File[]) {
+  async function handleImageUpload(files: File[]) {
+    const totalFiles = files.length + images.length;
+
```

Comment:
Please check if the file is a supported format, the drag and drop allows to me to attach unsupported file types